infra/gcp/terraform/k8s-infra-public-pii/main.tf

```diff
@@ -85,6 +85,25 @@ resource "google_storage_bucket" "audit-logs-gcs" {
   }
 }
 
+// service account for running ASN etl pipeline job
+resource "google_service_account" "asn_etl" {
+  project      = local.project_id
+  account_id   = "asn-etl"
+  display_name = "asn-etl"
+}
+
+// service account for running ASN etl pipeline job
+data "google_service_account" "public_pii_asn_etl" {
+  account_id = "asn-etl"
+  project    = local.project_id
+}
+
+// service account for running ASN etl pipeline job
+data "google_service_account" "ii_sandbox_asn_etl" {
+  account_id = "asn-etl"
+  project    = "k8s-infra-ii-sandbox"
+}
```

Comment:
Not needed. Only workloads or applications running inside this project are allowed to interact with BigQuery.

Reply:
Using this to refer to the account in the RoleBinding below